docs/debugging-dotnet-lambda-functions.md

```diff
@@ -0,0 +1,149 @@
+# Debugging .NET Core lambda functions
+
+These instructions outline how you can debug a lambda handler locally using the SAM CLI, and attach the VS Code debugger to it.
+
+## Install and configure prerequisites
+
+1. Install the [AWS Toolkit for Visual Studio Code](https://github.com/aws/aws-toolkit-vscode#getting-started).
+2. Install the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp). This extension gives VS Code the ability to debug .NET Core applications.
+3. Launch Visual Studio Code and open a SAM application or create a new one. <!-- TODO: Link to separate doc with instructions. -->
+
+    Note: Open the folder that contains `template.yaml`.
+
+4. Open a terminal in the folder containing `template.yaml` and set up the debugger by running the following commands:
+
+    * Replace `path/to/project` with the relative path to the directory containing the `.csproj` project that you want to debug.
```

Comment:
Consider using `${path_to_project}` or `<path_to_project>`

Reply:
I'll clean up replacement variables throughout this doc.

Reply:
✔️